### ISSUE TITLE
Implement Dark/Light Mode Toggle

### ISSUE THREAD

**Issue by Sangeeta-20-stack** [ShudarsanRegmi/ClassMgmtSysOS-#29]
Description:
The project currently does not have a theme toggle. Adding a dark/light mode toggle will improve accessibility and user experience, allowing users to switch between themes based on preference.

Expected Behavior:

Users can switch between dark and light modes using a toggle button.

The UI updates smoothly without breaking existing components.

User’s theme preference persists across sessions.

Kindly assign me this issue.

**Comment by zaidxGithub** [comment]
can you assign this issue to me, i would like to work on it @ShudarsanRegmi

**Comment by diksha78dev** [comment]
Hi , I am GSSOC'25 Contributor ,I would like to work on this issue. Can you please assign this issue to me?
